src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlInternalTransaction.cs

```diff
@@ -361,9 +309,11 @@ private int GetServerTransactionLevel()
                 SqlParameter parameter = new SqlParameter("@out", SqlDbType.Int);
```

Comment:
NIT: Could you move the inner comments at lines 301 to 303 on the XML description?